packages/shared/src/components/modals/common/Modal.tsx

```diff
@@ -52,7 +52,7 @@ const modalKindToClassName: Record<ModalKind, string> = {
     'h-full max-h-[calc(100vh-2.5rem)] mobileL:h-[40rem] mobileL:max-h-[calc(100vh-5rem)]',
   [ModalKind.FlexibleCenter]:
     'mx-4 max-w-[calc(100vw-2rem)] max-h-[min(calc(100vh),40rem)] mobileL:max-h-[min(calc(100vh-5rem),40rem)]',
-  [ModalKind.FlexibleTop]: 'max-h-full h-full mobileL:h-auto',
+  [ModalKind.FlexibleTop]: 'max-h-full h-auto',
```

Comment:
I think this might reintroduce some scroll issues again
Meaning if you now scroll inside the modal, down-up then close it the backdrop will scroll on the feed.

Could you please double check that first?